app/src/main/res/values/strings.xml

```diff
@@ -153,6 +155,7 @@ to your home internet.</string>
   <string name="enable_remaining_sensors">Enable %1$d Sensors</string>
   <string name="enabled_summary">When enabled values will be sent to Home Assistant</string>
   <string name="enabled_title">Enabled</string>
+  <string name="enabled">Enabled</string>
```

Comment:
should we just stick to using `enabled_title` since the string is the same?

Reply:
When i think about that now, i would use only one. If the strings need to be different, than we can still change this.